### ISSUE TITLE
Error when requesting an url with double slashes, right after hostname

### ISSUE THREAD

**Issue by tpoindessous** [containous/traefik#1323]
### What version of Traefik are you using (`traefik version`)?
1.1.2

### What is your environment & configuration (arguments, toml...)?
I have an example toml, which uses Google as an backend :
    url = "https://www.google.com:443"

### What did you do?
First, I tested my conf : http://localhost/images/branding/googlelogo/2x/googlelogo_color_120x44dp.png 
It worked.

Then, I tried http://localhost//images/branding/googlelogo/2x/googlelogo_color_120x44dp.png
It didn't work, I got this error from Google : 
The requested URL /branding/googlelogo/2x/googlelogo_color_120x44dp.png was not found on this server

It seems that Traefik removed "/images/" from the URL.

If I tried another url with double slashed but inside the url, it worked : http://localhost/images/branding//googlelogo/2x/googlelogo_color_120x44dp.png

### What did you expect to see?
These two URL should work in a identical way : 
http://localhost//images/branding//googlelogo/2x/googlelogo_color_120x44dp.png
http://localhost/images/branding/googlelogo/2x/googlelogo_color_120x44dp.png

**Comment by kurlzor** [comment]
I have the same problem. I use Jenkins behind Traefik and some pictures don't show up because of this.
Example: https://jenkins.host//static/5a36b3a5/images/48x48/user.png
Result (notice the `//static/` part is missing): 
```
HTTP ERROR 404

Problem accessing /5a36b3a5/images/48x48/user.png. Reason:

    Not Found
Powered by Jetty://
```
If I remove that extra slash, it works. It also works if I bypass traefik completely and request that same URL to Jenkins directly.

**Comment by SeMeKh** [comment]
Same story with Jenkins. Trying to pinpoint the bug, I found out that the container is getting a request of the form
`GET http://api/json HTTP/1.1`
instead of
`GET //api/json HTTP/1.1`

**Comment by andrewcfitz** [comment]
@kurlzor does your login page work?

**Comment by kurlzor** [comment]
@andrewcfitz Yep, it does. I answered in #1431.

**Comment by hrach** [comment]
@ldez 
I see you have marked this as duplicate, closed the duplicated issue and set the milestone to 1.3. 
I've tried 1.3.0-rc1 but it doesn't seem to be fixed there.

**Comment by ldez** [comment]
@hraf it's not fixed in 1.3, sorry for the confusion, I added the label on the 2 issues instead of 1. 😉

**Comment by oliviermadre** [comment]
Same problem here :

```
1/ Single Slash
URL in chrome :
http://myhost/someroute/a/b/c/d

URL as seen in nginx (debug mode) just behind traefik :
2017/05/30 13:33:40 [debug] 6#0: *79 http request line: "GET /someroute/a/b/c/d HTTP/1.1"
2017/05/30 13:33:40 [debug] 6#0: *79 http uri: "/someroute/a/b/c/d"


2/ Double Slash
URL in chrome :
http://myhost//someroute/a/b/c/d

URL as seen in nginx (debug mode) just behind traefik :
2017/05/30 13:35:46 [debug] 6#0: *81 http request line: "GET http://someroute/a/b/c/d HTTP/1.1"
2017/05/30 13:35:46 [debug] 6#0: *81 http uri: "/a/b/c/d"
```

**Comment by ldez** [comment]
Do you still have the same problem with the latest version? [releases](https://github.com/containous/traefik/releases)

#1699 can resolve this problem.

**Comment by hrach** [comment]
@ldez it seems it's not fixed in 1.3.0. Do you mean any other relase? Actually, the PR was merged just 3 days ago and the 1.3.0 is relased about 2 weeks.

**Comment by ldez** [comment]
it's a mistake: the good PR #1638

**Comment by hrach** [comment]
Ok :) But it still seems it doesnt work.

**Comment by kjvalencik** [comment]
Just tested the latest build and it still an issue.

**Comment by fabiorauber** [comment]
Tested with traefik:1.3.3-alpine and it is still an issue. Just found another hosted site that suffers from this, not being able to load a script.

**Comment by dtomcej** [comment]
In case anyone was wondering, double slashes are allowed in paths via [RFC 3986 ](http://www.ietf.org/rfc/rfc3986.txt). I suggest we add a unit test to ensure that this case gets tested.

**Comment by andrewcfitz** [comment]
Any updates?

**Comment by opskumu** [comment]
Any updates?

**Comment by ldez** [comment]
We understand the importance of the subject.
We will try to work on it as soon as possible.
You're welcome to submit a PR if we don't already created one.

Remember the gentle way to participate:
- add a "reaction" on the first message of the issue.
- add more useful and detailed information on the subject.
- resolve the issue by making a PR.

**Comment by soedar** [comment]
For those affected by this bug, please try out traefik:v1.5.0-rc1 and see if that had fixed the issue for you.

**Comment by hrach** [comment]
Confirming it seems to be fixed! Thanks! 🎉

**Comment by ldez** [comment]
Closed by  #2451
